### ISSUE TITLE
Battle for the Undercity

### ISSUE THREAD

**Issue by Tritonek** [dalaranwow/dalaran-wow#5188]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Hello, so there was this gnome mage "Meixu" who was doing the quest "Battle for the Undercity". While doing the the quest, someone tried to kill him (I tried to do it later on too), but the buff from the quest made it impossible. His health regen was incredible. I even talked to one of the GM's (Aesiris) who explained it to me.
**Expected behaviour**: Tell us what should happen instead
Even with the buff he should be "able" to die.
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![wowscrnshot_121717_142020](https://user-images.githubusercontent.com/18738446/34080052-bfc4c2d6-e338-11e7-80fc-63ad7f3955db.jpg)
![gm chat](https://user-images.githubusercontent.com/18738446/34080055-c3ac95c2-e338-11e7-8fc8-60c67eaf4a41.jpg)

**Comment by TeridanDW** [comment]
Thank you for the effort @Tritonek, but this report contains no information for the issue itself only the result.
If there is more information please post below.
